src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0, 0 because the value is less than 15`, () => {
```

Comment:
Not "because". And not "value".
Please make a better description of your tests. Here is an example:
`should return 0 for cat and dog when they are less than 15 years old`